sdk/deviceupdate/azure-iot-deviceupdate/README.md

```diff
@@ -69,17 +69,19 @@ catch (HttpResponseException ex) {
 
 ## Next steps
 
-Get started with our [Device Update for IoT Hub samples](./src/samples)
+Get started with our [Device Update for IoT Hub samples](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/deviceupdate/azure-iot-deviceupdate/src/samples)
 
 ## Contributing
 
-This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit <https://cla.microsoft.com.>
+This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
-This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
```

Comment:
I don't see links for `coc`, `coc_faq`, or `coc_contact`

Reply:
I am surprised this wasn't caught by the link verification tool. 
cc: @sima-zhu

Reply:
It is not a link to link-checker script.

We have converter which converts markdown to html, and then parse the http link in html.
In this case, github did not recognize this as a link, so it skips the checking.
In markdown,
![image](https://user-images.githubusercontent.com/48036328/110048198-3ce65000-7d04-11eb-80cf-33b76e64b602.png)

In source code:
![image](https://user-images.githubusercontent.com/48036328/110048397-a36b6e00-7d04-11eb-86b2-bbb793c5f022.png)

Reply:
Markdowns allow adding links in multiple ways, so, maybe we need to extend the tooling to check these as well.